redisinsight/ui/src/pages/rdi/statistics/StatisticsPage.tsx

```diff
@@ -0,0 +1,126 @@
+import { isEmpty } from 'lodash'
+import React, { useEffect } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { useParams } from 'react-router-dom'
+
+import { connectedInstanceSelector, fetchConnectedInstanceAction } from 'uiSrc/slices/rdi/instances'
+import { fetchRdiPipeline, rdiPipelineSelector } from 'uiSrc/slices/rdi/pipeline'
+import { fetchRdiStatistics, rdiStatisticsSelector } from 'uiSrc/slices/rdi/statistics'
+import { TelemetryEvent, TelemetryPageView, sendEventTelemetry, sendPageViewTelemetry } from 'uiSrc/telemetry'
+import { ExplorePanelTemplate } from 'uiSrc/templates'
+import { formatLongName, setTitle } from 'uiSrc/utils'
+import Clients from './clients'
+import DataStreams from './data-streams'
+import Empty from './empty'
+import RdiStatisticsHeader from './header'
+import ProcessingPerformance from './processing-performance'
+import Status from './status'
+import TargetConnections from './target-connections'
+
+import styles from './styles.module.scss'
+
+const StatisticsPage = () => {
+  const { rdiInstanceId } = useParams<{ rdiInstanceId: string }>()
+
+  const dispatch = useDispatch()
+
+  const { data: pipelineData } = useSelector(rdiPipelineSelector)
+  const { loading: isStatisticsLoading, data: statisticsData } = useSelector(rdiStatisticsSelector)
+  const { name: connectedRdiInstanceName } = useSelector(connectedInstanceSelector)
+
+  const rdiInstanceName = formatLongName(connectedRdiInstanceName, 33, 0, '...')
+  setTitle(`${rdiInstanceName} - Pipeline Status`)
+
+  const onRefresh = (section: string) => {
+    dispatch(fetchRdiStatistics(rdiInstanceId, section))
+  }
+
+  const onRefreshClicked = (section: string) => {
+    sendEventTelemetry({
+      event: TelemetryEvent.RDI_STATISTICS_REFRESH_CLICKED,
+      eventData: {
+        rdiInstanceId,
+        section
+      }
+    })
+  }
+
+  const onChangeAutoRefresh = (section: string, enableAutoRefresh: boolean, refreshRate: string) => {
+    sendEventTelemetry({
+      event: enableAutoRefresh
+        ? TelemetryEvent.RDI_STATISTICS_AUTO_REFRESH_ENABLED
+        : TelemetryEvent.RDI_STATISTICS_AUTO_REFRESH_DISABLED,
+      eventData: {
+        rdiInstanceId,
+        section,
+        enableAutoRefresh,
+        refreshRate
+      }
+    })
+  }
+
+  useEffect(() => {
+    if (!pipelineData) {
```

Comment:
probably you can move fetch rdi pipeline and fetchConnectedInstanceAction in Instance page and remove it from here and from pipelineMangementPage, what do you think about it?

Reply:
are we worried about page refreshes?

Reply:
good idea